internal/typeinfo/validate.go

```diff
@@ -45,7 +45,7 @@ func ValidateOutputs(args []any) (TypeToValue, error) {
 	typeToValue := TypeToValue{}
 	for _, arg := range args {
 		v := reflect.ValueOf(arg)
-		if isNil(v) {
+		if isInvalidNil(v) {
```

Comment:
Instead of keeping the boolean signature, make it return an error. That way, you can return a message based on the case you detect, then just return it if it's not `nil`. Call it `ValidateValue`.

Reply:
Changed type of function and made error messages more useful.